CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 2.17.0
+
+### Added
+
+* add experimental support for reviewed preprints in press packages in `application/vnd.elife.press-package+json;version=4`
+
+### Changed
+
+* removed experimental flag from `application/vnd.elife.press-package+json;version=3`
+
+### Removed
+
+* removed support for `application/vnd.elife.press-package+json;version=1`
+* removed support for `application/vnd.elife.press-package+json;version=2`
```

Comment:
It's unusual to remove support for both previous versions (1, 2) and introduce two new versions (3, 4). There is no room to transition for services that depend on those content type versions.

I'd rather we support three versions (2, 3, 4) and mark the older two as deprecated (2, 3), following up with a separate PR to remove the oldest (2) after the downstream updates are complete.

I'd say that if we could absolutely guarantee nothing is using press-packages v1 or v2 then go for it, but obviously `journal` and `api-sdk-php` at the very least are using it.